app/composables/useDataHover.ts

```diff
@@ -44,6 +53,27 @@ export const useDataHover = () => {
       return `${sign}${delta}s`
     }
 
+    // Calculate delta for Dates
+    if (hoveredRawValue instanceof Date && currentRawValue instanceof Date) {
+      const deltaMs = currentRawValue.getTime() - hoveredRawValue.getTime()
+      if (deltaMs === 0) return null
+
+      // Convert to seconds and use formatDuration for human-readable format
+      const deltaSeconds = Math.abs(deltaMs) / 1000
+      const sign = deltaMs > 0 ? '+' : '-'
+
+      // Use formatDuration from date-fns for human-readable format
+      const d = new Date() // arbitrary date
+      const humanDuration = formatDuration(
+        intervalToDuration({
+          start: d,
+          end: new Date(d.getTime() + deltaSeconds * 1000),
+        }),
+      )
```

Comment:
am I tripping or is this overthinking it? why not just pass the two actual `Date`s to the date-fns method instead of going through all this indirection?